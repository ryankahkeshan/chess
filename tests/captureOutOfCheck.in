game human human
move e2 e4
move d7 d5
move e4 d5
move c7 c5
move d5 c6
move d8 d7
move c6 d7
move c8 d7
