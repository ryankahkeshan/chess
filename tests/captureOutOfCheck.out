8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _P_ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 ppp pppp
6  _ _ _ _
5 _ _p_ _ 
4  _ _P_ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 ppp pppp
6  _ _ _ _
5 _ _P_ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 pp_ pppp
6  _ _ _ _
5 _ pP_ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 pp_ pppp
6  _P_ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh

8 rnb_kbnr
7 pp_qpppp
6  _P_ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh

8 rnb_kbnr
7 pp_Ppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh

Black is in check.
8 rn _kbnr
7 pp_bpppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh

Final Score:
White: 0
Black: 0
